srcpkgs/python3-ipython_ipykernel/template

```diff
@@ -1,19 +1,24 @@
 # Template file for 'python3-ipython_ipykernel'
 pkgname=python3-ipython_ipykernel
-version=5.5.4
-revision=3
-build_style=python3-module
-hostmakedepends="python3-setuptools python3-ipython
- python3-jupyter_client python3-traitlets python3-tornado"
-depends="python3-ipython python3-traitlets python3-jupyter_client python3-tornado"
+version=6.19.2
+revision=1
+build_style=python3-pep517
+hostmakedepends="hatchling python3-jupyter_client python3-jupyter_core
+ python3-platformdirs"
+depends="python3-ipython python3-traitlets python3-jupyter_client python3-tornado
+ python3-matplotlib-inline python3-nest_asyncio python3-packaging python3-psutil
+ python3 python3-comm python3-pyzmq"
 short_desc="IPython kernel for Jupyter (Python3)"
 maintainer="Alessio Sergi <al3hex@gmail.com>"
 license="BSD-3-Clause"
 homepage="https://github.com/ipython/ipykernel"
 distfiles="${PYPI_SITE}/i/ipykernel/ipykernel-${version}.tar.gz"
-checksum=1ce0e83672cc3bfdc1ffb5603e1d77ab125f24b41abc4612e22bfb3e994c0db2
-# checks create a cycle with python3-jupyter_core and python3-jupyter_client
-make_check=no
+checksum=1ab68d3d3654196266baa93990055413e167263ffbe4cfe834f871bcd3d3506d
+make_check=no # cycle with python3-jupyter_core and python3-jupyter_client
+
+do_install() {
+	python3 -m installer --destdir ${DESTDIR} ${make_install_args} ${make_install_target} dist/*.whl
+}
```

Comment:
```suggestion
make_install_target="dist/ipython_ipykernel-${version}-*-*-*.whl"
```

Reply:
Another untested suggestion. The actual value for `make_install_target` should be

```bash
make_install_target="dist/ipykernel-${version}-*-*-*.whl"
```

Sorry for the noise....

Reply:
done.